### ISSUE TITLE
PgUp and PgDn buttons behave unexpectedly

### ISSUE THREAD

**Issue by Photon89** [xournalpp/xournalpp#1412]
**Affects versions :**
 - OS: Manjaro Linux
 - Version of Xournal++: 1.0.13

**Describe the bug**
The default behaviour of the "page up" and "page down" buttons is to skip the height of one screen size. This way, when you are reading something and reach the bottom of the page, after pressing "page down" you can continue reading at the line where you stopped.

In Xournal, however, these buttons are mapped to "next page" and "previous page". This way, if your zoom level is not exactly "fit height", you jump from some position at a given page to the top of the next page, skipping parts of the page which did not fit onto the screen.

**To Reproduce**
1. Load any PDF.
2. Choose a sufficiently high zoom factor (say, "fit width").
3. Press "page down" on your keyboard.
4. The top of the next page will be on the top of your screen.

**Expected behavior**
The bottom of the screen should jump to the top of the screen as it is the case in most other applications.

**Comment by Technius** [comment]
As noted in #2063, the page up/down should go down by 0.8-0.9 page increments.

**Comment by VivekThazhathattil** [comment]
This issue can still be reproduced in the latest version (1.1.0). Is this issue still relevant? Can I work on it?

**Comment by LittleHuba** [comment]
You can of course work on this issue. Please use `release-1.1`as base branch for your work.

**Comment by hyperupcall** [comment]
I don't think this is still an issue? Page Up/Down skips the height of a little less than one screen size and doesn't skip any lines. As shown below:

![Peek 2022-10-16 23-42](https://user-images.githubusercontent.com/24364012/196106773-7eabc8de-3c62-46bb-8ee1-fbb069e1bdd6.gif)

Is this a stale and already fixed issue that can be closed?

**Comment by Photon89** [comment]
For me (version 1.1.1, Linux) it is still an issue. Which version are you using?

**Comment by hyperupcall** [comment]
> For me (version 1.1.1, Linux) it is still an issue. Which version are you using?

Thanks for the reply - This is on the latest version. I'll check that version too and see what's up.

**Comment by Photon89** [comment]
1.1.2 landed in Arch Linux recently and it seems to be fixed there! I am waiting for it to land in Manjaro which I use on my laptop with stylus to test it there, but I don't expect it to behave it any differently than on Arch.

**Comment by Photon89** [comment]
On Manjaro the problem persists. Possibly something configuration related?

**Comment by Photon89** [comment]
@bhennion I still see this issue on latest git, the bug is not fixed for me!

**Comment by Photon89** [comment]
Thanks for reopening! Is there anything I can do to help track down the issue?

**Comment by Photon89** [comment]
Still an issue in 1.2.0. Also tested with empty config folder `~/.config/xournalpp`, so it is not a configuration issue.
